tests/drivers/can/api/src/canfd.c

```diff
@@ -413,6 +477,29 @@ ZTEST_USER(canfd, test_set_bitrate_too_high)
 	zassert_equal(err, 0, "failed to start CAN controller (err %d)", err);
 }
 
+/**
+ * @brief Test setting max supported data phase bitrate.
+ */
+ZTEST_USER(canfd, test_set_data_bitrate_max_fd)
```

Comment:
Positive test.
API states that can_get_bitrate_max() returns "Maximum supported bitrate in bits/s"
In this test I check that value returned by can_get_bitrate_max() is accepted by can_set_bitrate_data().